RecoEgamma/EgammaTools/interface/EgammaLocalCovParamDefaults.h

```diff
@@ -0,0 +1,10 @@
+#ifndef RecoEgamma_EgammaTools_interface_EgammaLocalCovParamDefaults_h
+#define RecoEgamma_EgammaTools_interface_EgammaLocalCovParamDefaults_h
+
+struct EgammaLocalCovParamDefaults {
+  static constexpr float kRelEnCut = 4.7;
+  static constexpr float kMultThresEB = 1.0;
+  static constexpr float kMultThresEE = 1.25;
```

Comment:
For the constants that may change during the lifetime of the experiment, I'm wondering if it would not be appropriate to make them configurable as the module parameters, with the defaults being set in fillDescriptions? I think you can still define the numbers here in this struct (to have the values in a single place across multiple classes), but just to avoid having to recompile the code in case these need to be changed for some reason.

Reply:
Yes, now I have made multThresEB & multThresEE configurable, as there is a chance that they might change when ECAL DPG comes up with new set of PF recHit thresholds. The other parameter, kRelEnCut = 4.7, is not configurable as this parameter is not expected to change.